tsconfig.json

```diff
@@ -11,7 +11,8 @@
     "sourceMap": true,
     "strict": true,
     "types": [],
-    "verbatimModuleSyntax": true
+    "verbatimModuleSyntax": true,
+    "skipLibCheck": true
```

Comment:
**Nitpick:** I'm always anxious about using `skipLibCheck`. It essentially means some lib has bad typing and is often a symptom of a forgotten update, or a bug to report upstream.

Reply:
Is it due to https://github.com/vitejs/vite/issues/9813 ?

Reply:
I removed the `skipLibCheck` and applied the workaround mentioned in the issue.